File: 241008/ab 사이에 있는 c/c-between-a-and-b.py
abc = input()
arr = abc.split()
a = int(arr[0])
b = int(arr[1])
c = int(arr[2])

satisfied = False

for i in range (a, b + 1):
     if (i % c == 0):
         satisfied = True

if (satisfied == True):
     print("YES")
else:
     print("NO")

File: 241008/ab 사이에 있는 c/c-between-a-and-b.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split(" ");

const a = Number(input[0]);
const b = Number(input[1]);
const c = Number(input[2]);

let satisfied = false;

for (let i = a; i <= b; i++){
    if (i % c === 0) {
        satisfied = true;
    } 
}

if (satisfied === true) {
    console.log("YES");
} else {
    console.log("NO");
}